Userland/DevTools/UserspaceEmulator/SoftCPU.cpp

```diff
@@ -1834,7 +1834,18 @@ void SoftCPU::FNDISI(const X86::Instruction&) { TODO_INSN(); }
 void SoftCPU::FNCLEX(const X86::Instruction&) { TODO_INSN(); }
 void SoftCPU::FNINIT(const X86::Instruction&) { TODO_INSN(); }
 void SoftCPU::FNSETPM(const X86::Instruction&) { TODO_INSN(); }
-void SoftCPU::FLD_RM80(const X86::Instruction&) { TODO_INSN(); }
+
+void SoftCPU::FLD_RM80(const X86::Instruction& insn)
+{
+    VERIFY(!insn.modrm().is_register());
+
+    // long doubles can be up to 128 bits wide in memmory for reasons (alignment) and only uses 80 bits of precision
+    // bit_cast seems to see the 12 bytes, but godbolt says 16 bytes in x86-64 or 12 with -m32
+    // and writing a 12 byte read just for this would be ineficient
```

Comment:
ineficient -> inefficient
Also, why would it be inefficient?

Reply:
code wise  
it would be another function that would have to be implemented across multiple classes like the other reads/writes  
in a previous comment I have described a way to do it in 2 writes dependent on long double size
In the newer version its described and handled better